app/serializers/comment_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class CommentSerializer < ActiveModel::Serializer
   embed :ids, include: true
-  attributes :id, :body, :mentioned_usernames, :created_at, :updated_at, :parent_id, :parent_author_name
+  attributes :id, :body, :mentioned_usernames, :created_at, :updated_at, :parent_id, :parent_author_name, :edited
```

Comment:
In a complete change of opinion, I think we should serialize out the versions count, rather that just a boolean, which will enable this functionality down the road:

![screen shot 2016-02-11 at 1 26 23 pm](https://cloud.githubusercontent.com/assets/750477/12966428/13ad5924-d0c3-11e5-8a0a-bbdd2c9d4652.png)

(ie... it's been edited, but how many times!?)